slitlessutils/examples/wr96.py

```diff
@@ -46,6 +47,21 @@
 SUFFIX, DRZSUF = 'flc', 'drc'
 SCALE = 0.05          # driz image pix scale
 
+WR96_DRIZZLE_PARAMS = {
```

Comment:
I don't understand what these are for?  This looks to override the drizzling for the direct image. If so, then I don't think we should use your method for that.  Either way, I don't think we should expose these parameters as a dict here.  Because there are multiple types of drizzling, and it will get confusing.  Can we keep this dict in the function it goes with "preprocessing_direct()" to avoid confusion?  For example, some of these will be different for grism data (like final_rot and final_scale, and driz_cr and so on).